### DISCUSSION TITLE
Cannot remove values from event stream without breaking pipeline stream

### DISCUSSION THREAD

**Discussion by flomickl** [apache/streampipes#1276] [Dev]
### Body

I tried to remove some values from the stream using the removeFieldBySelector

```
 event.removeFieldBySelector(latitudeMapper);
 event.removeFieldBySelector(longitudeMapper);
```

Full code here:
https://github.com/apache/streampipes/blob/8d0aea2a4bb9c9736512e527a199796ae25071ec/streampipes-extensions/streampipes-processors-geo-jvm/src/main/java/org/apache/streampipes/processors/geo/jvm/jts/processor/latlngtojtspoint/LatLngToJtsPointProcessor.java#L105

The values are removed from the stream, but something unknown, strange happens.
The next processor will not receive any events!

Is something wrong with the output strategy???

Workflow and example data can be found here
https://cwiki.apache.org/confluence/x/QZBbDg
and uncomment this lines. 

### StreamPipes Committer

I acknowledge that I am a maintainer/committer of the Apache StreamPipes project.

**Comment by bossenti** [comment]
@flomickl this sounds for me more like a discussion
If you don't agree, feel free to revert it to an issue

**Comment by flomickl** [comment]
so do I have to adjust the output strategy other than .keep() or append()  if I use
`event.removeFieldBySelector`

```
...
public void onInvocation ...
        .outputStrategy(
            OutputStrategies.append(
                PrimitivePropertyBuilder
                    .create(Datatypes.String, GEOMETRY_RUNTIME)
                    .domainProperty("http://www.opengis.net/ont/geosparql#Geometry")
                    .build()
            )
        )
...

public void onEvent

...
event.removeFieldBySelector(latitudeMapper);
event.removeFieldBySelector(longitudeMapper);
 event.addField(GEOMETRY_RUNTIME, geom.toString());
...
```
